.travis.yml

```diff
@@ -45,6 +45,6 @@ jobs:
         docker exec -it "bats-dest-tests" bash -c 'bats -r -f "destructive-." .'
 # for now spellcheck tests are not enforced, return true to consider result as ok
     - stage: Shellcheck
-      script: shellcheck -x -s bash openhabian-setup.sh || true
-    - script: shellcheck -x -s bash functions/*.bash || true
-    - script: shellcheck -x -s bash build-image/*.bash || true
+      script: shellcheck -x -s bash openhabian-setup.sh
+    - script: shellcheck -x -s bash functions/*.bash
+    - script: shellcheck -x -s bash build-image/*.bash
```

Comment:
maybe the more general approach would be better, i.e. do it per default for any .bash / ,sh file in the repo.
This could by done by running a openhabian-shellcheck.bash script which does a find over the repo and calls shellcheck accordingly for each file except the dashboard-include sh. preferrably with an output in the log for each checked file.
(find . -iname "*.sh" && find . -iname "*.bash")|grep -v dashboard)... 
But we can change this later. Please do not block this merge by introducing this now.

Reply:
Ok I'll merge like this. I wouldn't agree anyway as only a build with NO shellsheck errors whatsoever is a good build, and any author should know how to manually shellcheck his changes anyway